File: mobile/src/main/java/com/alexstyl/specialdates/upcoming/widget/list/NoAds.java
package com.alexstyl.specialdates.upcoming.widget.list;

import com.alexstyl.specialdates.upcoming.UpcomingEventsAdRules;

class NoAds implements UpcomingEventsAdRules {

    @Override
    public boolean shouldAppendAdForEndOfMonth(int index) {
        return false;
    }

    @Override
    public void onNewMonthAdded(int index) {
        // do nothing
    }

    @Override
    public boolean shouldAppendAdAt(int index) {
        return false;
    }

    @Override
    public void onNewAdAdded(int index) {
        // do nothing
    }
}

File: android_mobile/src/main/java/com/alexstyl/specialdates/upcoming/widget/list/NoAds.kt
package com.alexstyl.specialdates.upcoming.widget.list

import com.alexstyl.specialdates.upcoming.UpcomingEventsAdRules

class NoAds : UpcomingEventsAdRules {

    override fun shouldAppendAd(): Boolean {
        return false
    }

    override fun onNewAdAdded() {
        // do nothing
    }
}
